src/main/java/dev/revere/alley/config/ConfigServiceImpl.java

```diff
@@ -56,9 +56,23 @@ public void setup(AlleyContext context) {
             this.loadConfig(fileName);
         }
 
+        this.createSchematicsFolder();
         this.assignConfigs();
     }
 
+    private void createSchematicsFolder() {
+        File file = new File(this.plugin.getDataFolder(), "schematics");
+        if (!file.exists()) {
+            if (file.mkdirs()) {
+                Logger.info("Created schematics folder: " + file.getPath());
+            } else {
+                Logger.error("Failed to create schematics folder: " + file.getPath());
+            }
+        } else {
+            Logger.info("Schematics folder already exists: " + file.getPath());
```

Comment:
This message is redundant. We shouldn't need to know if it already exists or not as that's the core functionality. Rather only log messages where errors occur, not normal behavior.